data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg

```diff
@@ -372,6 +373,7 @@
         [message]
             speaker=narrator
             message= _ "A monolith of pale stone, luminous and bright, but cold and brittle to the touch."
+            image=wesnoth-icon.png
```

Comment:
Unless it's specified, these should all be image=none.

Reply:
by "these should all" do you mean all the monoliths, all the instances in DiD, or all instances in the entire PR?

Reply:
Everything in DiD. If it doesn't specify an image in the original, assume it's none.

Reply:
ok, I'm pretty sure I replaced all the icons introduced by 0165e25 with 488dc65; there were some instances where the icon was already there beforehand that I left alone, though.